Makefile

```diff
@@ -85,6 +85,11 @@ test-all:
 	@echo Running all tests
 	$(GO) test -mod=vendor -race -v -tags 'unit e2e' $(GO_PACKAGES)
 
+.PHONY: coverage
+coverage:
+	$(GO) test -mod=vendor -race -tags unit -coverprofile=coverage.txt ./...
```

Comment:
I think it's fine to only use unit test coverage as it should be the same as the `e2e` one

Reply:
I would add both tags to the coverage, as there are things that we are only testing in `e2e` (the client creation and connection for example)

Reply:
The issue with also running the e2e test is that they (have to) run in docker. Which makes it a bit hard to extract the coverage report. Would you be fine just running the unit tests for now and I open a ticket for the e2e tests?

Reply:
I think that is a good idea, opening another ticket to refine it later.